### ISSUE TITLE
gcc warning: format ‘%x’ expects argument of type ‘unsigned int’

### ISSUE THREAD

**Issue by slayoo** [gnudatalanguage/gdl#345]
```
envt.cpp:1325:44: warning: format ‘%x’ expects argument of type ‘unsigned int’, but argument 4 has type ‘const void*’ [-Wformat=]
     ix,static_cast <const void *>(env[ix]) );
                                            ^
```

**Comment by slayoo** [comment]
(e.g. here: https://travis-ci.org/slayoo/gdl/jobs/392357355)

**Comment by rexso** [comment]
Fixed in #346
